core/features/environment.py

```diff
@@ -0,0 +1,34 @@
+from selenium import webdriver
+
+BEHAVE_DEBUG_ON_ERROR = True
+
+
+def setup_debug_on_error(userdata):
+    global BEHAVE_DEBUG_ON_ERROR
+    BEHAVE_DEBUG_ON_ERROR = userdata.getbool("BEHAVE_DEBUG_ON_ERROR")
+
+
+def before_all(context):
+
+    setup_debug_on_error(context.config.userdata)
+
+    desired_capabilities = webdriver.DesiredCapabilities.FIREFOX
+    desired_capabilities['version'] = '12'
+    desired_capabilities['platform'] = 'WINDOWS'
+    desired_capabilities['name'] = 'Testing Col-League App'
+    desired_capabilities['client_key'] = '44fa53ac66807797ca8ec3f1e9f9a9d9'
+    desired_capabilities['client_secret'] = '5cc0cce8a93b22b347ffd72204d007bb'
```

Comment:
It is the beginning creation of behave. It is client keys to the behave testing site I setup. It doesnt do anything yet but I wanted to get some stuff down